MorkoUnityProject/Assets/Scripts/FieldOfView.cs

```diff
@@ -7,20 +7,29 @@
 /// </summary>
 public class FieldOfView : MonoBehaviour
 {
+    [Range(-1f,1f)]
+    public float hitOffset;
+
     public float viewRadius;
     [Range(0,360)]
     public float viewAngle;
 
     public float maskCutDistance = 0.2f;
 
     public MeshFilter viewMeshFilter;
+    public MeshFilter fullMeshFilter;
     Mesh viewMesh;
+    Mesh fullMesh;
 
     private void Start()
     {
         viewMesh = new Mesh();
         viewMesh.name = "View Mesh";
         viewMeshFilter.mesh = viewMesh;
+
+        fullMesh = new Mesh();
+        fullMesh.name = "Full Mesh";
+        fullMeshFilter.mesh = fullMesh;
     }
 
     public Vector3 DirFromAngle(float degreeAngle, bool globalAngle)
```

Comment:
change `globalAngle` to `useGlobalAngle` or similar to communicate its actual meaning